tests/appsec/test_extended_data_collection.py

```diff
@@ -45,6 +45,13 @@
 )
 
 
+def assert_body_property(body, prop, expected_value) -> None:
+    if context.library.name == "nodejs":
```

Comment:
Doesn't it make sense to have an if/else covering the rest of libraries?

Reply:
Yes!